metecho/api/views.py

```diff
@@ -228,6 +230,38 @@ def review(self, request, pk=None):
         )
         return Response(self.get_serializer(task).data, status=status.HTTP_202_ACCEPTED)
 
+    @action(detail=True, methods=["POST"])
+    def can_reassign(self, request, pk=None):
+        serializer = CanReassignSerializer(data=request.data)
+        if not serializer.is_valid():
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+        task = self.get_object()
+        role = serializer.validated_data["role"]
+        role_org_type = {
+            "assigned_qa": SCRATCH_ORG_TYPES.QA,
+            "assigned_dev": SCRATCH_ORG_TYPES.Dev,
+        }.get(role, None)
+        gh_uid = serializer.validated_data["gh_uid"]
+        # We want to consider soft-deleted orgs, too:
+        org = task.scratchorg_set.all().filter(org_type=role_org_type).first()
```

Comment:
@wlonk In this case don't we specifically only care about active orgs? That's why we can do `.first()` instead of iterating over all orgs -- this endpoint is only "can the _current active org_ for this task-role be reassigned to this new user?"